README.md

```diff
@@ -6,7 +6,17 @@ GLB GELF is a Go library used for structured log messages generation. They could
 Installation
 ----------
 
+Using version 1.0.0
+
+```shell
+go get github.com/Graylog2/go-gelf/gelf
+go get github.com/globocom/glbgelf
+```
+
+Using version 2.0.0
```

Comment:
Hmm... I don't understand that, we really want to support two versions of glbgelf? I think isn't necessary.

Reply:
You're right. This is unnecessary! I'll take this part off and leave the version 2 support only.

Reply:
Done! The correction was made on [this commit](https://github.com/globocom/glbgelf/pull/6/commits/bf03eb8ecfb7009259f83419ca4dea025e7f076b)